src/modules/shared/layouts/AppLayout/AppLayout.tsx

```diff
@@ -84,7 +84,9 @@ const AppLayout: FC = ({ children }) => {
 	const windowSize = useWindowSize();
 	const isMobile = !!(windowSize.width && windowSize.width < Breakpoints.xxl);
 	const showBorder = useSelector(selectShowNavigationBorder);
-	const { data: accessibleVisitorSpaces } = useGetAccessibleVisitorSpaces();
+	const { data: accessibleVisitorSpaces } = useGetAccessibleVisitorSpaces({
+		canViewAllSpaces: user?.permissions.includes(Permission.READ_ALL_SPACES) ?? false,
+	});
```

Comment:
You can use the existing useHasAllPermission hook.

```suggestion
        const canViewAllSpaces = useHasAllPermission(Permission.READ_ALL_SPACES);
	const { data: accessibleVisitorSpaces } = useGetAccessibleVisitorSpaces({
		canViewAllSpaces
	});
```